apps/core/src/pages/auth.tsx

```diff
@@ -128,7 +173,13 @@ export const Component = () => {
   }
 
   if (loginStatus === 'authenticated') {
-    return <AuthPage />;
+    return (
+      <>
+        <AuthPage />
+        {runtimeConfig.enableNotificationCenter && <NotificationCenter />}
```

Comment:
it seems to me this should be put at the app root and remove the one in workspace layout, regardless of user's login status.